lib/request.js

```diff
@@ -92,9 +92,14 @@ function Request (options) {
 
   const parsedURL = parseURL(options.url || options.path, options.query)
 
+  /**
+   * https://github.com/nodejs/node/blob/75b68c6e4db515f76df73af476eccf382bbcb00a/lib/_http_common.js
+   */
   this.url = parsedURL.pathname + parsedURL.search
 
-  this.httpVersion = '1.1'
+  this.httpVersionMajor = 1
+  this.httpVersionMinor = 1
+  this.httpVersion = `${this.httpVersionMajor}.${this.httpVersionMinor}`
```

Comment:
```suggestion
  this.httpVersion = '1.1'
```

Reply:
Is there any difference?

Reply:
If you look at the code in https://github.com/nodejs/node/blob/75b68c6e4db515f76df73af476eccf382bbcb00a/lib/_http_common.js#L94 then you see that httpVersionMajor and httpVersionMinor are variable

But in our code here httpVersionMajor and httpVersionMinor are always set to 1. So we can save us some cycles to build the string and directly set it to `1.1`

Reply:
fixed